policyengine_us/variables/gov/simulation/labor_supply_response/labor_supply_response.py

```diff
@@ -0,0 +1,127 @@
+from policyengine_us.model_api import *
+
+
+class relative_income_change(Variable):
+    value_type = float
+    entity = Person
+    label = "relative income change"
+    unit = "/1"
+    definition_period = YEAR
+    requires_computation_after = "labor_supply_response"
+
+    def formula(person, period, parameters):
+        simulation = person.simulation
+        measurement_branch = simulation.get_branch("lsr_measurement")
+        baseline_branch = simulation.get_branch("baseline").get_branch(
+            "baseline_lsr_measurement"
+        )
+        measurement_person = measurement_branch.populations["person"]
+        baseline_person = baseline_branch.populations["person"]
+        baseline_net_income = baseline_person.household(
+            "household_net_income", period
+        )
+        net_income = measurement_person.household(
+            "household_net_income", period
+        )
+        baseline_net_income_c = np.where(
+            baseline_net_income == 0, 1, baseline_net_income
+        )
+        net_income_c = np.where(net_income == 0, 1, net_income)
+        relative_change = (
+            net_income_c - baseline_net_income_c
+        ) / baseline_net_income_c
+        return np.clip(relative_change, -1, 1)
+
+
+class relative_wage_change(Variable):
+    value_type = float
+    entity = Person
+    label = "relative wage change"
+    unit = "/1"
+    definition_period = YEAR
+    requires_computation_after = "labor_supply_response"
+
+    def formula(person, period, parameters):
+        simulation = person.simulation
+        measurement_branch = simulation.get_branch("lsr_measurement")
+        baseline_branch = simulation.get_branch("baseline").get_branch(
+            "baseline_lsr_measurement"
+        )
+        measurement_person = measurement_branch.populations["person"]
+        baseline_person = baseline_branch.populations["person"]
+        baseline_mtr = baseline_person("marginal_tax_rate", period)
+        baseline_wage = 1 - baseline_mtr
+        mtr = measurement_person("marginal_tax_rate", period)
+        wage = 1 - mtr
```

Comment:
I think `wage` implies currency
```suggestion
        net_marginal_wage_rate = 1 - mtr
```

Reply:
This treats 100% MTR as 0% MTR, which results in us predicting that, due to substitution effects, someone going from 100% MTR to 1% MTR will lower labor supply (and vice versa).

How about instead we set 100% MTR to 99%?

Reply:
Line 57 set 1 to 0.01